packages/server/lib/controllers/apiAuth.controller.ts

```diff
@@ -28,10 +28,11 @@ import type { MessageRowInsert } from '@nangohq/types';
 
 class ApiAuthController {
     async apiKey(req: Request, res: Response<any, Required<RequestLocals>>, next: NextFunction) {
-        const { account, environment, authType } = res.locals;
+        const { account, environment } = res.locals;
         const { providerConfigKey } = req.params;
         const receivedConnectionId = req.query['connection_id'] as string | undefined;
         const connectionConfig = req.query['params'] != null ? getConnectionConfig(req.query['params']) : {};
+        const isConnectSession = res.locals['authType'] === 'connectSession';
```

Comment:
`isConnectSession` could almost be an attribute of `RequestLocals`

Reply:
maybe indeed